duckplayer/duckplayer-api/src/main/java/com/duckduckgo/duckplayer/api/DuckPlayer.kt

```diff
@@ -184,6 +184,20 @@ interface DuckPlayer {
      */
     fun shouldOpenDuckPlayerInNewTab(): OpenDuckPlayerInNewTab
 
+    /**
+     * Checks whether Duck Player should show a custom error view based on RC flag
+     *
+     * @return True if should show a custom error view, false otherwise.
+     */
+    fun shouldShowCustomError(): Boolean
```

Comment:
Similar to what I said above, we shouldn't expose this on the public API, but rather rely on CSS to get this values internally from the RC JSON